relay-server/src/services/store.rs

```diff
@@ -199,6 +202,17 @@ impl StoreService {
             KafkaTopic::Attachments
         } else if event_item.as_ref().map(|x| x.ty()) == Some(&ItemType::Transaction) {
             KafkaTopic::Transactions
+        } else if event_item.as_ref().map(|x| x.ty()) == Some(&ItemType::UserReportV2) {
+            let feedback_ingest_topic_rollout_rate = self
+                .global_config
+                .current()
+                .options
+                .feedback_ingest_topic_rollout_rate;
+            if is_rolled_out(scoping.organization_id, feedback_ingest_topic_rollout_rate) {
```

Comment:
This is checking the rollout rate with the org, not the event -- let's please roll out slowly.

Reply:
I think it's just used as a boolean on/off in practice, so should be fine 👍

Reply:
Not sure what the feedback event volume is, but suddenly going from 0 to N may cause a backlog until consumers scale up.

Reply:
Is there a function I can use to do it on the event level?

Reply:
We will probably keep it at the org level since feedback volumes aren't that high, and we'd like to test it on all feedback in s4s. Will comment here if that changes. Holding off merging until https://getsentry.atlassian.net/browse/OPS-5543 is done.

Reply:
> We will probably keep it at the org level since feedback volumes aren't that high

That's totally fine, my comment was just a note to roll out slowly to avoid any surprises 😄